packages/material-ui/src/ButtonGroup/ButtonGroup.js

```diff
@@ -55,20 +55,31 @@ export const styles = theme => ({
     boxShadow: 'none',
     '&:not(:last-child)': {
       borderRight: `1px solid ${theme.palette.grey[400]}`,
+      '&$disabled': {
+        borderRight: `1px solid ${theme.palette.action.disabled}`,
```

Comment:
Why not use JSS structure?
ex:
```border: [[1, 'solid', 'red']]```

Reply:
I didn't know about the JSS structure :D I don't see it used anywhere else in the codebase, but if it's preferred, I can always switch it to that

Reply:
@oliviertassinari which one is better in the codebase?

Reply:
JSS has a plugin to enable this syntax. We don't include it by default to minimize the bundle size. Unfortunately, it's not available. Also, notice that we will most likely write the styles in CSS in the future #6115, leaving the JavaScript syntax aside.